src/option/none.ts

```diff
@@ -70,5 +70,5 @@ export class NoneT<T> implements OptT<T> {
  * @returns {OptT<T>}
  */
 export function getNone<T>(): OptT<T> {
-  return new NoneT();
```

Comment:
not holding the review up, but I'm curious as to why you made this change.

Did TS complain about it?  Or is this a readability thing?

Reply:
I'm sorry, I didnt really mean to commit that - I was getting some errors when running tsc and that was helping - but then I realized that I was running an older version of typescript, updating it fixed the issue - I'll send another PR to remove it.